base/tuple.jl

```diff
@@ -615,3 +615,37 @@ empty(@nospecialize x::Tuple) = ()
 
 foreach(f, itr::Tuple) = foldl((_, x) -> (f(x); nothing), itr, init=nothing)
 foreach(f, itrs::Tuple...) = foldl((_, xs) -> (f(xs...); nothing), zip(itrs...), init=nothing)
+
+"""
```

Comment:
Perhaps we may merge the tuple/array docstrings, as it feels like the methods are intrinsically similar?

Reply:
Hello @jishnub. Thank you for this comment. For tuples, we only support circular shift in the first dimension. On the other hand, the array version supports a shift in each dimension. I feel that merging the documentation might confuse the users. What do you feel?

Reply:
I think that they're sufficiently similar to be merged, especially as even in the array case, specifying `shift` as an integer only shifts the first dimension. I believe the behavior for vectors and tuples would be identical?

Reply:
Thank you for the comment, @jishnub. Indeed, if `shifts` is a scalar, then the behaviour is similar. However, if we merge the docs, then we will also have the signature `circshift(x, shifts)` for the tuple case. I felt that with such a signature, users will expect support for multi-dimensional shifts also for tuples, which would be wrong. However, since you're well versed with the other docs, please let me know what's the best way forward and I'll just go ahead and implement that.

Reply:
Thank you, @jishnub for your kind reply and the link to the documentation. I will merge the documentation, as suggested, at the earliest.